app/index.html

```diff
@@ -0,0 +1,38 @@
+<html>
+  <head>
+  <title>juice-box</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="stylesheets/screen.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  </head>
+
+<body>
+
+<div class="container">
+  <div class="head">
+    <h1>juice-box-draw</h1>
+  </div>
+
+<div class="inline-head">
+  <div class="prompt">
+    <h2>prompt</h2>
+  </div>
+
+  <div id="timer" class="timer">
+    <h2>timer</h2>
+    <span id="minutes"></span>
+    <span id="colon">:</span>
+    <span id="seconds"></span>
+  </div>
+</div>
+
+  <div class="canvas">
+    <canvas id="juice-canvas" width="100" height="100"></canvas>
+  </div>
+
+</div>
+
+  <script src="js/app.js"></script>
+
+</body>
+</html>
```

Comment:
End file with blank line, and let's improve the indents & blank lines throughout this file